### DISCUSSION TITLE
Will it work with regular River (no Max or Pro)

### DISCUSSION THREAD

**Discussion by ast0815** [vwt12eh8/hassio-ecoflow#47] [Q&A] [answered]
Hi, this integration looks very cool!

The readme says the integration is compatible with River Max and Pro. Should it also work with the regular River? Or has this just never been tested? The regular River and the Pro/Max versions look very similar, so I would assume that it should work, but it would be a shame if I bought it just to find out it does not integrate with my HA.

**Comment by jiriteach** [comment, marked as answer]
It does work with the regular River which is the just River Pro without the extra battery. I had these originally and worked no problem.

**Reply by ast0815** [reply]
Thanks!
